gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,656 @@
++
+[source]
+----
+https://jsonplaceholder.typicode.com/
+----
+.. Type `80` in the *Port* field.
+.. Click *Next* to open the *Review* page.
+
+. In the *Review* page associated with *APIs / Add API*: 
+.. Review your API configuration.
+.. If necessary, use *Edit* to revise any configurations. 
+.. When your configuration is correct, click *Save & Deploy*.
++
+Inside the API instance, all HTTP requests to the external IP associated with your gateway are proxied to the `jsonplaceholder` service.
+. From a terminal window, test the API instance:
+.. Get your external IP address and use it to check that your IP redirects to the *Implementation URI*:
++
+[source,kubectl]
+----
+kubectl -n gateway get services
+----  
++
+The command returns a result that looks similar to this example:
++
+[[get-services]]
+----
+NAME    TYPE         CLUSTER-IP     EXTERNAL-IP       PORT(S)                    AGE
+ingress LoadBalancer 10.100.145.109 <123456.aws.com>  80:30524/TCP,443:31710/TCP 7d7h
+----
++
+The `EXTERNAL-IP` value `<123456.aws.com>` is an example and not an actual value.
++
+.. Run the following `curl` command, replacing `_your_external_ip_` with _your_ `EXTERNAL-IP` from the preceding step:
++
+[source,curl]
+----
+curl -s -o /dev/null -w "%{http_code}\n" --request GET 'http://_your_external_ip_:80/users'
+----
++
+The command executes a `GET` request to the API and prints a success code:
++
+----
+200
+----
++
+You can also run a simpler `curl` command to return an array of JSON objects from the service's `/users` endpoint, for example: 
++
+[source,curl]
+----
+$ curl http://_your_external_ip_:80/users
+----  
+
+== Step 6: Secure Your API Using the Basic Authentication Policy
+
+Use API Manager to add a basic authentication policy to your API.
+
+. In *API Manager*, select *API Administration* from the side navigation panel.
+. Find and click the name of the API that you created in <<create-api>>, such as *my-api*.
+. Navigate to the *Policies* page for your API:
+.. Select *Policies* from the side navigation panel.
+.. In the *Policies* page, click *Add policy*.
++
+image::gateway-quickstart-add-policy.png["Add Policy page with Add policy button",width=90%]
+. In the page that opens, find the *Security* policy called *Basic Authentication - Simple* policy.
++
+image::gateway-quickstart-k8-basic-auth.png["Screenshot of 'Add a policy' page",width=90%]
++
+Be careful to avoid selecting the similarly named policy for LDAP.
+. Click *Next*.
+. Configure a Basic Authentication policy:
++
+image::gateway-quickstart-config-basic-auth.png["Screenshot of 'Configure Basic Authentication - Simple policy' page",width=80%]
++
+.. For *User Name*, enter `user` instead of your real user name. 
++
+If the field is pre-populated with a value, you can keep the value or change it if you need to protect it.
++
+.. For *User Password*, enter `pw`.
++
+You are not required to change this value if you do not need to protect it.
```

Comment:
I'm not sure what this means, what do you mean with `if you do not need to protect it`?